mmv1/products/bigquery/Job.yaml

```diff
@@ -600,6 +608,22 @@ properties:
                 description: |
                   Describes the Cloud KMS encryption key version used to protect destination BigQuery table.
                 output: true
+          - !ruby/object:Api::Type::NestedObject
+            name: 'parquetOptions'
+            description: |
+              Parquet Options for load and make external tables.
+            properties:
+              - !ruby/object:Api::Type::Boolean
+                name: 'enumAsString'
+                description: |
+                  If sourceFormat is set to PARQUET, indicates whether to infer Parquet ENUM logical type as STRING instead of BYTES by default.
+              - !ruby/object:Api::Type::Boolean
+                name: 'enableListInference'
+                description: |
+                  If sourceFormat is set to PARQUET, indicates whether to use schema inference specifically for Parquet LIST logical type.
+                at_least_one_of:
+                  - configuration.0.load.0.parquet_options.0.enum_as_string
```

Comment:
If this field is a boolean, why do we have an `at_least_one_of` here?

Reply:
I thought `at_least_one_of`  would make sure the object parquet_options is not empty.